cli/webhook.py

```diff
@@ -0,0 +1,124 @@
+import sys
+import slackweb
+from sets import Set
+from datetime import datetime
+from cli.settings import Settings
+from cli.appconfig import AppConfig
+
+
+class WebHook:
+
+    flag = False
+
+    def __init__(self):
+        '''this flag makes sure if initialization fails, many hook
+        steps wont try to post message again and again '''
+        self.disabled = True
+        self.emoji = ':rocket:'
+        self.settingObj = Settings()
+        self.appconfigObj = AppConfig()
+
+    def webhookSetting(self):
+
+        config_dir = self.settingObj.getConfigDir()
+        roger_env = self.appconfigObj.getRogerEnv(config_dir)
```

Comment:
Could we take the 2 lines congif_dir and roger_env , within the "try" block. or was there any specific reason of not putting them in try block